### PR TITLE
chore: Change APK name in update-apk.sh

### LINKED ISSUE #1201: Change package name in upload-apk.sh
### Chore

**Description**
The package name and the name of generated apk needs to be changed in the upload-apk.sh.

**Would you like to work on the issue?**
Yes

### PR DESCRIPTION
**Pull request by sridharjajoo**
Fixes #1201 
Checklist:

- [ ] I have checked for PMD and check-style issues
- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream development branch.

Screenshots for the change:

### CODE REVIEW

**Comment by sridharjajoo** [comment]
@iamareebjamal From where can we change the package name in the script? There is a variable PACKAGE_NAME in the script but where does it reference to?

**Comment by iamareebjamal** [comment]
It's in Circle CI environment variables

File: scripts/update-apk.sh [hunk 1]
```diff
@@ -27,9 +27,11 @@ if [ "$TRAVIS_BRANCH" == "$PUBLISH_BRANCH" ]; then
 	${ANDROID_HOME}/build-tools/27.0.3/zipalign -v -p 4 app-release-unaligned.apk app-release.apk
 fi
 
-for file in app*; do
-  mv $file test-${file%%}
-done
+if [ "$TRAVIS_BRANCH" == "$PUBLISH_BRANCH" ]; then
+    for file in app*; do
+          cp $file eventyay-organizer-master-${file%%}
```
Comment (iamareebjamal) [inline scripts/update-apk.sh]: This won't do the trick. Check open-event-android and susi_android

**Comment by sridharjajoo** [comment]
@iamareebjamal Is the environment variable present in fossasia build environment like the places API?

**Comment by iamareebjamal** [comment]
I have changed the environment variable in Travis CI

File: scripts/update-apk.sh (hunk 1 above)
Comment (iamareebjamal) [reply scripts/update-apk.sh]: The fastlane app name below isn't changed

Comment (sridharjajoo) [reply scripts/update-apk.sh]: @iamareebjamal Changed the name.

Comment (sridharjajoo) [reply scripts/update-apk.sh]: should it be eventyay-organizer-master?

File: scripts/update-apk.sh [hunk 2]
```diff
@@ -53,4 +62,4 @@ if [ "$TRAVIS_BRANCH" != "$PUBLISH_BRANCH" ]; then
 fi
 
 gem install fastlane
-fastlane supply --apk test-app-release.apk --track alpha --json_key ../scripts/fastlane.json --package_name $PACKAGE_NAME
+fastlane supply --apk eventyay-organizer-app-release.apk --track alpha --json_key ../scripts/fastlane.json --package_name $PACKAGE_NAME
```
Comment (iamareebjamal) [inline scripts/update-apk.sh]: Wrong name here

Comment (sridharjajoo) [reply scripts/update-apk.sh]: So it should be eventyay-organizer-master right?

Comment (iamareebjamal) [reply scripts/update-apk.sh]: Yes

Comment (iamareebjamal) [reply scripts/update-apk.sh]: You can leave it to be app-release, it will work as well

**Comment by sridharjajoo** [comment]
I have changed the name to app-release. Kindly review and merge

**Pull request merged**